### DISCUSSION TITLE
N-ImageNet

### DISCUSSION THREAD

**Discussion by msjun23** [neuromorphs/tonic#222] [Ideas]
Hi! I found out interesting neuromorphic dataset, N-ImageNet made by [here](https://github.com/82magnolia/n_imagenet) and want to use it by awesome lib tonic.

As everyone knows, large dataset is very import topic in DL/ML and ImageNet really contributed many studies. In neuromorphic, event-based study, I think ImageNet could be a powerful dataset and N-ImageNet is just published last year.

So I want to ask that, have you any plan to or can you supply N-ImageNet by tonic? or is it in progress already?

**Comment by fabrizio-ottati** [comment]
Hi! I had a look at it and I discussed with @biphasic. We are more interested in having natively neuromorphic datasets but, if you would like to submit a PR with the code for N-ImageNet, we would be pretty happy to take a look at it. However, wait for @biphasic opinion about this.

**Reply by biphasic** [reply]
100% agreed with this :+1:
